src/app/main/main.html

```diff
@@ -60,6 +60,19 @@ <h1 id="folder-heading">Folder Settings</h1>
           </label>
         </fieldset>
 
+        <fieldset id="scanModeFieldset">
+          <label for="scan-mode">Scan mode:</label>
+          <select id="scan-mode" name="scan-mode">
+            <option value="anywhere">Anywhere in the page</option>
+            <option value="inside-elements">Inside the selected HTML
+              elements
+            </option>
+            <option value="count-only">That change the number of HTML elements
+              (content is ignored)
+            </option>
+          </select>
+          <output name="scan-mode-description"></output>
+        </fieldset>
 
```

Comment:
To be honest, the whole UI kinda needs an overhaul. The labels are very inconsistent, some are to the left some above. Scan mode is right below URL because it affects all the fields below it.

Reply:
Agreed! I'd still prefer the Scan Mode stuff to be all together at the bottom, even if the count mode causes the Change threshold to be disabled further up the page. The goal is to make it as clear as possible for people who don't understand or want to use these new settings.